### DISCUSSION TITLE
Suggestion : add a method to zoom on a given position

### DISCUSSION THREAD

**Discussion by mgermerie** [iTowns/itowns#1638] [Ideas]
Discussing with @ftoromanoff, we figured it could be interesting to add a method that users could call in order to zoom in a given position. This method - which we could name for instance `zoomInTowardPosition` - would take a world position as a parameter and would trigger a zoom in toward this given position when called. This would allow users to bind custom buttons to zoom actions.
An equivalent method would be added for zoom out.
Otherwise, we could add a single method which would take two parameters : a target position for the zoom and a number. According to the sign of the number, the performed zoom could be zoom in or zoom out.

I believe the adequate place for such a method would be in controls (mainly `PlanarControls` since the zoom in `GlobeControls` is toward the center of the screen, but perhaps we could implement it in globe too).

What do you think @gchoqueux ?

**Comment by gchoqueux** [comment]
[`CameraUtils.animateCameraToLookAtTarget`](http://www.itowns-project.org/itowns/docs/#api/Controls/CameraUtils) look like this feature?

**Reply by gchoqueux** [reply]
could you try `CameraUtils.getCameraTransformOptionsFromExtent`?
